src/control/server/storage/provider.go

```diff
@@ -99,6 +99,17 @@ func (p *Provider) ControlMetadataNeedsFormat() (bool, error) {
 	return p.metadata.NeedsFormat(req)
 }
 
+// ControlMetadataHasPath checks whether metadata section is defined
+func (p *Provider) ControlMetadataHasPath() bool {
```

Comment:
Naming suggestion - how about `ControlMetadataConfigured` to more accurately describe the intent of this method.